<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>change_Password</class>
 <widget class="QDialog" name="change_Password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family:century gothic;
	font-size:24px;
}
QPushButton
{
background-color:pink;
border-radius:30px;
}
QPushButton:hover
{
background:#49ebff

}
QLineEdit
{
background-color:#333;
color:white;
}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>650</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_cardnumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>491</width>
     <height>111</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>card number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_recent_pass">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>491</width>
     <height>111</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>last password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_new_pass">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>491</width>
     <height>111</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>new password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_verify_new_pass">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>491</width>
     <height>111</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>repeat new password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
